### DISCUSSION TITLE
求问大佬关于http/socks5代理的问题

### DISCUSSION THREAD

**Discussion by Rain-Swift** [vernesong/OpenClash#3690] [General]
如题，我在第一次设置了代理服务器和端口为openclash的后，弹出了认证窗口，然后关闭后，我如何才能再次打开这个认证窗口输入账号密码？

**Comment by ghost** [comment]
什么软件和版本号也说出来

这样知道的人看到了才能回你啊

**Reply by Rain-Swift** [reply]
最新版的openclash，用Chrome浏览器打开网页弹出的认证窗口

**Reply by ghost** [reply]
chrome?电脑版？
那就是windows的吧
这玩意不是报错后就自动弹出来吗？

你搜一下凭据管理器

错了的话就再等有缘人吧，没用过chrome
